src/mds/PurgeQueue.cc

```diff
@@ -376,9 +376,6 @@ uint32_t PurgeQueue::_calculate_ops(const PurgeItem &item) const
 
     ops_required = std::min(num, g_conf()->filer_max_purge_ops);
 
-    // Account for removing (or zeroing) backtrace
```

Comment:
Can you move this comment to line 374 above? The `: 1` is confusing otherwise.

Reply:
@batrick done.